src/coreclr/nativeaot/Runtime/StackFrameIterator.h

```diff
@@ -137,9 +137,11 @@ class StackFrameIterator
         // This is a state returned by Next() which indicates that we just unwound a reverse pinvoke method
         UnwoundReversePInvoke = 0x20,
 
+        IgnoreExCollide = 0x40,
```

Comment:
The ExCollide is not really ignored when this is set. It is respected by the stackwalker in general. It is just the one place that is asking for the colliding clause index that is a problem.

A better way to fix this may be to change arguments of `RhpSfiNext` to pointers, allow passing NULL, and avoid the problem with the colliding clause that way. What do you think?

Reply:
updated